1-js/06-advanced-functions/01-recursion/article.md

```diff
@@ -39,7 +39,7 @@ There are two ways to implement it.
     alert( pow(2, 3) ); // 8
     ```
 
-2. Recursive thinking: simplify the task and call self:
+2. Recursive thinking -- calls itself to simplify the task:
```

Comment:
```suggestion
2. Recursive thinking: calls itself to simplify the task:
```

Reply:
did you mean to review your own PR? you can commit directly to it, and it will automatically be added to the PR